auto-generated/vector-crypto/llvm-overloaded-tests/vsm4r.c

```diff
@@ -0,0 +1,47 @@
+// REQUIRES: riscv-registered-target
```

Comment:
I guess vsm4r should have both `vsm4r_vv` and `vsm4r_vs` for its overload intrinsics~